cluster/src/main/java/com/hartwig/pipeline/execution/vm/unix/GrepFileCommand.java

```diff
@@ -0,0 +1,20 @@
+package com.hartwig.pipeline.execution.vm.unix;
+
+import static java.lang.String.format;
+
+import com.hartwig.pipeline.execution.vm.BashCommand;
+
+public class GrepFileCommand implements BashCommand {
```

Comment:
GrepFileCommand could extend GrepCommand

Reply:
GrepCommand is no longer needed so removed.